gomega_dsl.go

```diff
@@ -125,7 +125,9 @@ func Ω(actual interface{}, extra ...interface{}) GomegaAssertion {
 //
 //Then:
 //    Expect(MyAmazingThing()).Should(Equal(3))
-//Will succeed only if `MyAmazingThing()` returns `(3, nil)`
+//Will succeed only if:
+//    MyAmazingThing()
+//    // Output: (3, nil)
```

Comment:
Is this a typo?

Reply:
I was trying to find the "go way" of documenting "this code yields this result" outside of "Examples" snippets. 

I could've sworn I saw this style on the go language docs somewhere, but now that I'm looking again I can't find it and I think I'm crazy and made this style up. I'll revert it back to how it was.

Reply:
This looks like the syntax for "runnable" examples :) I think the current sentence, it's clear enough though!